pkg/controller/controller.go

```diff
@@ -320,6 +321,16 @@ func (ctrl *resizeController) syncPVC(key string) error {
 		return fmt.Errorf("expected volume but got %+v", volumeObj)
 	}
 
+	if ctrl.isNodeExpandComplete(pvc, pv) && metav1.HasAnnotation(pv.ObjectMeta, util.AnnPreResizeCapacity) {
+		if err := ctrl.deletePreResizeCapAnnotation(pv); err != nil {
```

Comment:
how come resize controller is deleting the annotation? Shouldn't kubelet do it after nodeexpand is complete?

Reply:
This is because kubelet does not have the permission to modify PV objects.

Reply:
@gnufied is this an issue? Does this mean plugins that don't support controller resize and don't run the resizer will get this fix?

Reply:
Currently I think external-resizer is required even for plugins that don't support controller expansion, because in that case external-resizer calls a NO-OP control-plane expansion and successfully expands the PV. There is no way to update PV spec from kubelet otherwise. So every plugin that supports expansion needs to run resizer.